components/atoms/Title/Title.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useMemo } from 'react'
+import { StyleProp, StyleSheet, Text, TextProps, TextStyle } from 'react-native'
+import useTheme from '../../hooks/useTheme'
+
+export type HeadingVariation =
+  | 'heading1'
```

Comment:
@J-Rehan In slack I proposed to Saaniya that we rename this to `title1` since the overall component is called heading. Let's see what she says.